doc/mgr/dashboard_plugins/motd.inc.rst

```diff
@@ -3,34 +3,35 @@
 Message of the day (MOTD)
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Displays a configured `message of the day` at the top of the Ceph Dashboard.
+Displays a configured "message of the day" (MOTD) at the top of the Ceph
+Dashboard.
 
 The importance of a MOTD can be configured by its severity, which is
```

Comment:
an

Reply:
Fair enough. We'll go the whole hog.

Reply:
?  To be clear, the following line "of an MOTD"

Reply:
No, no. I caught it. The em-noise requires an "an" just as we would say "thine MOTD" instead of "thy MOTD" if we wrote in Early Modern English and we were inputing ownership of the MOTD to thee.